File: Converter10_CS/Model/SoruleModel.cs

namespace Converter10.Njc.Model
{

    #region 送金ルール基本情報モデル

    public class Sorule_Model
    {

        public string Vari_Sorule_guid { get; set; }
        public string Vari_Sorule_no { get; set; }
        public string Vari_Relation_guid { get; set; }
        public string Vari_Kubunsyo { get; set; }
        public string Vari_Sorule_startymd { get; set; }
        public string Vari_Sorule_endymd { get; set; }
        public string Vari_Kanri_keitaikbn { get; set; }
        public string Vari_Ikkatu_kanriflg { get; set; }
        public string Vari_Cyukai_kyflg { get; set; }
        public string Vari_Cyukai_koflg { get; set; }
        public string Vari_Cyukai_kaiflg { get; set; }
        public string Vari_Soymd1_gaitoukbn { get; set; }
        public string Vari_Soymd1_simekbn { get; set; }
        public string Vari_Soymd1_sokintukikbn { get; set; }
        public string Vari_Soymd1_sokinsimekbn { get; set; }
        public string Vari_Soymd2_gaitoukbn { get; set; }
        public string Vari_Soymd2_simekbn { get; set; }
        public string Vari_Soymd2_sokintukikbn { get; set; }
        public string Vari_Soymd2_sokinsimekbn { get; set; }
        public string Vari_Soymd3_gaitoukbn { get; set; }
        public string Vari_Soymd3_simekbn { get; set; }
        public string Vari_Soymd3_sokintukikbn { get; set; }
        public string Vari_Soymd3_sokinsimekbn { get; set; }
        public string Vari_Soymd4_gaitoukbn { get; set; }
        public string Vari_Soymd4_simekbn { get; set; }
        public string Vari_Soymd4_sokintukikbn { get; set; }
        public string Vari_Soymd4_sokinsimekbn { get; set; }
        public string Vari_Soymd5_gaitoukbn { get; set; }
        public string Vari_Soymd5_simekbn { get; set; }
        public string Vari_Soymd5_sokintukikbn { get; set; }
        public string Vari_Soymd5_sokinsimekbn { get; set; }
        public string Vari_Sosaki_multikbn { get; set; }
        public string Vari_Sosaki_koteiflg { get; set; }
        public string Vari_Sosaki_koteisu { get; set; }
        public string Vari_Sosaki_anbunflg { get; set; }
        public string Vari_Sosaki_hasuuketorisaki { get; set; }
        public string Vari_Sosaki_hasuadjustmentflg { get; set; }
        public string Vari_Ikkatu_kbn { get; set; }
        public string Vari_Ikkatu_bkgak { get; set; }
        public string Vari_Ikkatu_menseki { get; set; }
        public string Vari_Ikkatu_mensekimonth { get; set; }
        public string Vari_Kanritesu_kbn { get; set; }
        public string Vari_Kanritesu_cyosyukbn { get; set; }
        public string Vari_Kanri_reigaiky { get; set; }
        public string Vari_Kanri_reigaikai { get; set; }
        public string Vari_Hyteigaku_kbn { get; set; }
        public string Vari_Hyteigaku_kanrigak { get; set; }
        public string Vari_Hyteigaku_hiwariflg { get; set; }
        public string Vari_Bkteigaku_kanrigak { get; set; }
        public string Vari_Kanri_taxflg { get; set; }
        public string Vari_Biko_basic { get; set; }
        public string Vari_Sh_daihyohyflg { get; set; }
        public string Vari_Sh_kozabetuflg { get; set; }
        public string Vari_Sh_kozabetuListNokbn { get; set; }
        public string Vari_Hyikkatu_nkin_no1kbn { get; set; }
        public string Vari_Hyikkatu_nkin_no2kbn { get; set; }
        public string Vari_Hyikkatu_nkin_no3kbn { get; set; }
        public string Vari_Hyikkatu_nkin_no4kbn { get; set; }
        public string Vari_Hyikkatu_nkin_no5kbn { get; set; }
        public string Vari_Kanritesu_zeiumukbn { get; set; }
        public string Vari_History { get; set; }
        public string Vari_Rowid { get; set; }
        public string Vari_No_soruleflg { get; set; }
        public string Vari_Cyukai_zuijisokin { get; set; }
        public string Vari_Ikkatu_calckbn { get; set; }
        public string Vari_Ikkatu_taxflg { get; set; }
        public string Vari_Kanri_utizeiflg { get; set; }
        public string Vari_Soymd_basis { get; set; }

    }

    #endregion

    #region 送金ルール送金先情報モデル

    public class Sorule_sosaki_Model
    {

        public string Vari_Sorule_guid { get; set; }
        public string Vari_Sorule_no { get; set; }
        public string Vari_Sosaki_recno { get; set; }
        public string Vari_So_ow_no { get; set; }
        public string Vari_So_ow_kozano { get; set; }
        public string Vari_So_rit { get; set; }
        public string Vari_So_fixsogak { get; set; }

    }

    #endregion

    #region 送金ルール入金項目情報モデル

    public class Sorule_nk_cmrule_Model
    {

        public string Vari_Sorule_guid { get; set; }
        public string Vari_Sorule_no { get; set; }
        public string Vari_Taisyokbn { get; set; }
        public string Vari_Nkin_sortorder { get; set; }
        public string Vari_Nkin_no { get; set; }
        public string Vari_Sokin_rit { get; set; }
        public string Vari_Kanrigak_rit { get; set; }
        public string Vari_Hosyo_flg { get; set; }
        public string Vari_So_no { get; set; }

    }

    #endregion

    #region 送金ルール控除項目情報モデル

    public class Sorule_kojo_cmrule_Model
    {

        public string Vari_Sorule_guid { get; set; }
        public string Vari_Sorule_no { get; set; }
        public string Vari_Kojo_taisyokbn { get; set; }
        public string Vari_Nkin_sortorder { get; set; }
        public string Vari_Taisyonkin_no { get; set; }
        public string Vari_Sosai_flg { get; set; }
        public string Vari_Kojocalc_kbn { get; set; }
        public string Vari_Kojo_gak { get; set; }
        public string Vari_Kojo_gakzeikbn { get; set; }
        public string Vari_Kojonkin_no { get; set; }
        public string Vari_Kojo_rit { get; set; }
        public string Vari_Kojo_ritzeikbn { get; set; }
        public string Vari_Kojo_ritutizei { get; set; }
        public string Vari_Zei_rit { get; set; }
        public string Vari_Tateazu_flg { get; set; }
        public string Vari_Kojo_zeigak { get; set; }
        public string Vari_Sotaisyo_flg { get; set; }
        public string Vari_Sotaisyonkin_no { get; set; }

    }

    #endregion

}

File: Converter10_VB/Model/SoruleModel.vb
Namespace Njc.Model

#Region "送金ルール基本情報モデル"

    Public Class Sorule_Model

        Public Property Vari_Sorule_guid As String
        Public Property Vari_Sorule_no As String
        Public Property Vari_Relation_guid As String
        Public Property Vari_Kubunsyo As String
        Public Property Vari_Sorule_startymd As String
        Public Property Vari_Sorule_endymd As String
        Public Property Vari_Kanri_keitaikbn As String
        Public Property Vari_Ikkatu_kanriflg As String
        Public Property Vari_Cyukai_kyflg As String
        Public Property Vari_Cyukai_koflg As String
        Public Property Vari_Cyukai_kaiflg As String
        Public Property Vari_Soymd1_gaitoukbn As String
        Public Property Vari_Soymd1_simekbn As String
        Public Property Vari_Soymd1_sokintukikbn As String
        Public Property Vari_Soymd1_sokinsimekbn As String
        Public Property Vari_Soymd2_gaitoukbn As String
        Public Property Vari_Soymd2_simekbn As String
        Public Property Vari_Soymd2_sokintukikbn As String
        Public Property Vari_Soymd2_sokinsimekbn As String
        Public Property Vari_Soymd3_gaitoukbn As String
        Public Property Vari_Soymd3_simekbn As String
        Public Property Vari_Soymd3_sokintukikbn As String
        Public Property Vari_Soymd3_sokinsimekbn As String
        Public Property Vari_Soymd4_gaitoukbn As String
        Public Property Vari_Soymd4_simekbn As String
        Public Property Vari_Soymd4_sokintukikbn As String
        Public Property Vari_Soymd4_sokinsimekbn As String
        Public Property Vari_Soymd5_gaitoukbn As String
        Public Property Vari_Soymd5_simekbn As String
        Public Property Vari_Soymd5_sokintukikbn As String
        Public Property Vari_Soymd5_sokinsimekbn As String
        Public Property Vari_Sosaki_multikbn As String
        Public Property Vari_Sosaki_koteiflg As String
        Public Property Vari_Sosaki_koteisu As String
        Public Property Vari_Sosaki_anbunflg As String
        Public Property Vari_Sosaki_hasuuketorisaki As String
        Public Property Vari_Sosaki_hasuadjustmentflg As String
        Public Property Vari_Ikkatu_kbn As String
        Public Property Vari_Ikkatu_bkgak As String
        Public Property Vari_Ikkatu_menseki As String
        Public Property Vari_Ikkatu_mensekimonth As String
        Public Property Vari_Kanritesu_kbn As String
        Public Property Vari_Kanritesu_cyosyukbn As String
        Public Property Vari_Kanri_reigaiky As String
        Public Property Vari_Kanri_reigaikai As String
        Public Property Vari_Hyteigaku_kbn As String
        Public Property Vari_Hyteigaku_kanrigak As String
        Public Property Vari_Hyteigaku_hiwariflg As String
        Public Property Vari_Bkteigaku_kanrigak As String
        Public Property Vari_Kanri_taxflg As String
        Public Property Vari_Biko_basic As String
        Public Property Vari_Sh_daihyohyflg As String
        Public Property Vari_Sh_kozabetuflg As String
        Public Property Vari_Sh_kozabetuListNokbn As String
        Public Property Vari_Hyikkatu_nkin_no1kbn As String
        Public Property Vari_Hyikkatu_nkin_no2kbn As String
        Public Property Vari_Hyikkatu_nkin_no3kbn As String
        Public Property Vari_Hyikkatu_nkin_no4kbn As String
        Public Property Vari_Hyikkatu_nkin_no5kbn As String
        Public Property Vari_Kanritesu_zeiumukbn As String
        Public Property Vari_History As String
        Public Property Vari_Rowid As String
        Public Property Vari_No_soruleflg As String
        Public Property Vari_Cyukai_zuijisokin As String
        Public Property Vari_Ikkatu_calckbn As String
        Public Property Vari_Ikkatu_taxflg As String
        Public Property Vari_Kanri_utizeiflg As String
        Public Property Vari_Soymd_basis As String

    End Class

#End Region

#Region "送金ルール送金先情報モデル"

    Public Class Sorule_sosaki_Model

        Public Property Vari_Sorule_guid As String
        Public Property Vari_Sorule_no As String
        Public Property Vari_Sosaki_recno As String
        Public Property Vari_So_ow_no As String
        Public Property Vari_So_ow_kozano As String
        Public Property Vari_So_rit As String
        Public Property Vari_So_fixsogak As String

    End Class

#End Region

#Region "送金ルール入金項目情報モデル"

    Public Class Sorule_nk_cmrule_Model

        Public Property Vari_Sorule_guid As String
        Public Property Vari_Sorule_no As String
        Public Property Vari_Taisyokbn As String
        Public Property Vari_Nkin_sortorder As String
        Public Property Vari_Nkin_no As String
        Public Property Vari_Sokin_rit As String
        Public Property Vari_Kanrigak_rit As String
        Public Property Vari_Hosyo_flg As String
        Public Property Vari_So_no As String

    End Class

#End Region

#Region "送金ルール控除項目情報モデル"

    Public Class Sorule_kojo_cmrule_Model

        Public Property Vari_Sorule_guid As String
        Public Property Vari_Sorule_no As String
        Public Property Vari_Kojo_taisyokbn As String
        Public Property Vari_Nkin_sortorder As String
        Public Property Vari_Taisyonkin_no As String
        Public Property Vari_Sosai_flg As String
        Public Property Vari_Kojocalc_kbn As String
        Public Property Vari_Kojo_gak As String
        Public Property Vari_Kojo_gakzeikbn As String
        Public Property Vari_Kojonkin_no As String
        Public Property Vari_Kojo_rit As String
        Public Property Vari_Kojo_ritzeikbn As String
        Public Property Vari_Kojo_ritutizei As String
        Public Property Vari_Zei_rit As String
        Public Property Vari_Tateazu_flg As String
        Public Property Vari_Kojo_zeigak As String
        Public Property Vari_Sotaisyo_flg As String
        Public Property Vari_Sotaisyonkin_no As String

    End Class

#End Region

End Namespace
